### DISCUSSION TITLE
can't import github repository

### DISCUSSION THREAD

**Discussion by codelife123** [vercel/community#2531] [Bug Report]
I can't import any of my repository.
when I click import button it gives me this error 

![image](https://github.com/vercel/community/assets/87173258/2adf3633-b699-4871-89e5-ddfc7bd88f3e)

I have imported projects before this so I don't think it's something related permissions and it's not specific to one repo as well.
any idea ?

**Comment by codelife123** [comment]
seems it's not related to github import, I can't even clone existing template as well.

**Comment by amyegan** [comment]
Hi @codelife123. If you're still running into this error, please share the templates or public repos so I can look into the cause 🙏
